docs/source/user-guide/install/index.rst

```diff
@@ -2,10 +2,6 @@
 Installation
 ============
 
-.. contents::
-   :local:
-   :depth: 1
-
```

Comment:
The removal of the TOCs here is going to be replaced by one in the sidebar?

Reply:
Yes. I've gone through and delete all of these page TOCs because they are redundant.